libpgmodeler_ui/src/modelwidget.h

```diff
@@ -329,6 +329,12 @@ class ModelWidget: public QWidget {
 		//! \brief Returns the reference database model
 		DatabaseModel *getDatabaseModel(void);
 
+		//! \brief Returns the object scene used by the model
+		ObjectsScene *getObjectScene(void)const{return scene;};
```

Comment:
Just a note... I like methods constructions in the .cpp file even if they have a single line of code. Just a matter of taste. You can leave this one like this... I can fix later.

Reply:
Ok noted